### PR TITLE
feature/example integration with page parameters

### PR DESCRIPTION
**Pull request by stepp**
An example integration that uses the new page parameters feature.  This relates to the code in Tree repo PR #20 that adds parameters to integration pages and links:

https://github.com/treelabs/tree/pull/20

This PR should not be merged until Tree PR #20 is merged.

### CODE REVIEW

File: links/README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Tree Integration: Links
 
 A [UI Hook](https://treedocs.now.sh/docs/v1/hooks/ui/introduction/) serving a page that [links](https://treedocs.now.sh/docs/v1/blocks/basic/#link) to another page.
+We provide two examples: an `index.js` demonstrating basic links, and an `index2.js` that shows links with parameters.
```
Comment (michaelfester) [inline links/README.md:4]: I'd put everything in `index.js`. Page 1 has two links: the first is a simple link to Page 2, the second is a link to Page 3 with params.

Comment (stepp) [reply links/README.md]: Got it; fixed!

File: links/index.js [hunk 2]
```diff
@@ -42,31 +78,72 @@ var page2 = {
...
+            value: 'Back',
+            attrs: {
+                pageId: '1'
+            }
         }
     ]
 };
 
+// Page 3 uses two parameters for an image name and URL.
+var makePage3 = (params) => ({
```
Comment (michaelfester) [inline links/index.js:93]: Can we pass the parameters explicitly here (image_name and image_url), and leave it to the controller to break them up?

File: links/index.js [hunk 3]
```diff
@@ -1,16 +1,22 @@
+// An example of links between pages.
 var http = require('http');
+var querystring = require('querystring');
```
Comment (michaelfester) [inline links/index.js:3]: Should we update this example and remove query string and use `path`?

**Pull request closed without merging**